docs/getting-started-tutorial/1-view-first-development.adoc

```diff
@@ -0,0 +1,71 @@
+:idprefix:
+:idseparator: -
+:toc: right
+:toclevels: 2
+
+# View-first Development
+
+If you're developing a user-facing web site or application, one of Lift's
+greatest improvements over existing systems is view-first development.
+View-first development thoroughly separates the process of creating the user
+interface from the process of putting data from the system into it, in a way
+that lets you stay focused on users when you're creating the user interface and
+worry about the the interface between your backend and the HTML only when
+you're working on the backend.
```

Comment:
This might be easier to parse if it were broken into two sentences right around `in a way`

Reply:
Excellent point!